packages/node_modules/@webex/plugin-meetings/src/media/properties.js

```diff
@@ -25,7 +25,7 @@ export default class MediaProperties {
     this.remoteShare = options.remoteShare;
     this.remoteAudioTrack = options.remoteAudioTrack;
     this.remoteVideoTrack = options.remoteVideoTrack;
-    this.localQualityLevel = options.localQualityLevel || QUALITY_LEVELS.HIGH;
+    this.localQualityLevel = options.localQualityLevel || QUALITY_LEVELS['720p'];
     this.remoteQualityLevel = options.remoteQualityLevel || QUALITY_LEVELS.HIGH;
```

Comment:
As per discussion in our group it seems we need to make HIGH as 1080p and default should be 720p i.e. MEDIUM. Correct me if I misunderstood something

Reply:
Yes, @meta-dipanshu-sharma, low will be 480, Medium to 720p and high to 1080p. Remote we will be having as HIGH only, 
And Medium will be the default for local which is 720p.
@arun3528

Reply:
So, @meta-dipanshu-sharma As Arun also confirmed - Local 720p and remote is high (that's the current default)